ajax/libs/range.js/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "author": {
+    "email": "jscoderr@gmail.com",
+    "name": "Florian H."
+  },
+  "dependencies": {},
+  "description": "JavaScript's missing range function.",
+  "dist": {
+    "shasum": "bbe1f7e4044b9b1cfa2d0000dfb967651752497f",
+    "tarball": "http://registry.npmjs.org/range.js/-/range.js-1.1.0.tgz"
+  },
+  "homepage": "https://github.com/js-coder/range.js#readme",
+  "license": "MIT",
+  "name": "range.js",
+  "filename": "range.min.js",
+  "npmName": "range.js",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "**/*"
```

Comment:
It's not a good idea to use `"**/*"` in non-dist folder, please use `"range*"` instead.